syntax = "proto3";

package disparos;

service Disparo {

  rpc dimeCentroDianna (Vacio) returns (Distancia) {}

  rpc dispararCannon (InfoDisparo) returns (Distancia) {}

  rpc mejorDisparo (Vacio) returns (Usuario) {}
}

message Vacio{}

message Distancia {
    float distancia = 1;
}

message InfoDisparo {
    string usuario = 1;
    float angulo = 2;
    float velocidad = 3;
}

message Usuario {
    string usuario = 2;
}                         

